src/utils/utils.py

```diff
@@ -0,0 +1,7 @@
+def format_print(olist):
```

Comment:
Não estou tão certo quanto a adequação deste método. Entendo o que ele faz e o motivo, mas o modo como foi proposto, em um módulo útil, não parecer ser a melhor proposta. Dado que se o objetivo é apenas imprimir os objetos, então uma combinação entre __str__ nas classes de cada um e o print, e quando necessário o for para imprimir lista (conjuntos), já seria uma opção adequada e comum. Gostaria de entender melhor qual o motivo de optar fazer desse modo.